CMakeLists.txt

```diff
@@ -133,7 +133,8 @@ CONFIGURE_FILE(
 # Next, set up the testsuite
 
 SET(ASPECT_RUN_ALL_TESTS OFF CACHE BOOL "Set up complete test suite to run.")
-
+SET(ASPECT_COMPARE_TEST_RESULTS OFF CACHE BOOL "Compare test results with high accuracy.")
```

Comment:
while I wouldn't mind having this as the default, it means I have to update the tester before we can merge this PR.